packages/zoe/src/zoeService/zoeSeat.js

```diff
@@ -113,7 +113,7 @@ export const makeZoeSeatAdminFactory = baggage => {
         withdrawFacet,
         publisher,
         subscriber,
-        payouts: undefined,
+        payouts: {},
```

Comment:
this needs a `harden` (for failures https://github.com/Agoric/agoric-sdk/actions/runs/3745032447/jobs/6359022974 ) 

I'm force-pushing the fix so these commits can merge to master